src/status_im/chat/models/input.cljs

```diff
@@ -125,11 +126,15 @@
     (let [{:keys [message-id]}
           (get-in db [:chats current-chat-id :metadata :responding-to-message])
           show-name?     (get-in db [:multiaccount :show-name?])
-          preferred-name (when show-name? (get-in db [:multiaccount :preferred-name]))]
+          preferred-name (when show-name? (get-in db [:multiaccount :preferred-name]))
+          emoji? (message-content/emoji-only-content? {:text input-text
+                                                       :response-to message-id})]
       (fx/merge cofx
                 {:db (assoc-in db [:chats current-chat-id :metadata :responding-to-message] nil)}
                 (chat.message/send-message {:chat-id      current-chat-id
-                                            :content-type constants/content-type-text
+                                            :content-type (if emoji?
```

Comment:
How does this work in the context of mixed text/emoji?

Reply:
oh yeah good point, it gets rendered as massive text, probably best to validate that message, but will be in status-go, I am moving validation there, so I might do it in the scope of a separate PR